### PR TITLE
[hotfix][kafka][docs] Add warning regarding data losses when writing …

### PR DESCRIPTION
**Pull request by pnowojski**
This pull request add an alert in the documentation about possible data loses while writing to Kafka.

CC @tzulitai @aljoscha

### CODE REVIEW

File: docs/dev/connectors/kafka.md [hunk 1]
```diff
@@ -475,7 +475,13 @@ are other constructor variants that allow providing the following:
...
 ### Kafka Producers and Fault Tolerance
 
-With Flink's checkpointing enabled, the Flink Kafka Producer can provide
+#### Kafka 0.8
+
+Before 0.9 Kafka did not provide any mechanisms to guarantee at-least-once or exactly-once semantics.
+
+#### Kafka 0.9 and 0.10
+
+With Flink's checkpointing enabled, the `FlinkKafkaProducer010` can provide
```
Comment (aljoscha) [inline docs/dev/connectors/kafka.md]: Should this be `FlinkKafkaProducer09` and `FlinkKafkaProducer010`?

Comment (aljoscha) [reply docs/dev/connectors/kafka.md]: I can fix when merging.

**Comment by pnowojski** [comment]
@aljoscha fixed

**Comment by aljoscha** [comment]
Merged! 😃 

Could you please close this PR if it doesn't auto-merge?

**Comment by pnowojski** [comment]
Thanks :)

**Pull request closed without merging**